Makefile

```diff
@@ -21,7 +21,8 @@ test:
 	sbt test
 
 test-postgres:
-	docker run -d --name archway_postgres -p 5432:5432 -e POSTGRES_DB=archway postgres:9.3
+	docker network create archway
```

Comment:
This is good, eventually we will move this to Docker Compose and get it out of the Makefile